test/fixtures/poro.rb

```diff
@@ -309,11 +309,27 @@ class CachingCommentSerializer < CommentSerializer
     cache expires_in: 1.day, skip_digest: true
   end
 
-  class CachingModelSerializer < ModelSerializer
+  class CachingModelSerializer < ActiveModel::Serializer
```

Comment:
So, this bug 🐞 is essentially why all my benchmarks have shown a performance decrease until I memoized the associations? wow!